cmd/pg/wal_prefetch.go

```diff
@@ -17,12 +18,27 @@ var WalPrefetchCmd = &cobra.Command{
 	Args:   cobra.ExactArgs(2),
 	Hidden: true,
 	Run: func(cmd *cobra.Command, args []string) {
+		reconfigureLoggers()
+
 		folder, err := internal.ConfigureFolder()
 		tracelog.ErrorLogger.FatalOnError(err)
+
 		postgres.HandleWALPrefetch(internal.NewFolderReader(folder), args[0], args[1])
 	},
 }
 
 func init() {
 	Cmd.AddCommand(WalPrefetchCmd)
 }
+
+// wal-prefetch (WalPrefetchCmd) is internal tool, so to avoid confusion about errors in restoration process
+// we reconfigure loggers specially for internal use. All logs having PREFETCH prefix can be safely ignored
+func reconfigureLoggers() {
```

Comment:
Let's also redirect all loggers output to the `DebugLogger`. It will allow to debug wal-prefetch in case of issues but won't bloat the log in normal mode.